service/matching/forwarder.go

```diff
@@ -226,8 +226,7 @@ func (fwdr *Forwarder) ForwardPoll(ctx context.Context) (*internalTask, error) {
 			ForwardedFrom: &fwdr.taskListID.name,
 		})
 		if err != nil {
-			fwdr.handleErr(err)
-			return nil, err
+			return nil, fwdr.handleErr(err)
```

Comment:
This is not equivalent. The handleErr did the transfomation of error on purpose, I don't think we should return errForwarderSlowDown to API.  I think we want to ignore the error returned from handlerErr.